README.md

```diff
@@ -29,8 +29,9 @@
   </a>
 </p>
 
-neumorphic.css is a lightweight, responsive and modern CSS framework.
+neumorphic.css is a single-file, lightweight, responsive and modern CSS framework.
```

Comment:
Is it worth referencing this feature in the first line of the framework README? That line is supposed to give information regarding the framework in general, is including an installation method as a feature on that line a decent option?